libraries/core-react/stories/components/Menu.stories.tsx

```diff
@@ -34,11 +34,25 @@ Icon.add({
   cable,
 })
 
-const { MenuItem, MenuSection } = Menu
+const { Item, Section } = Menu
 const { Actions, Header } = TopBar
```

Comment:
Maybe change `Topbar` components to follow same dot annotation we talked about and remove deconstruct?

Reply:
Yes, updated all examples even those who was not actually changed. 👍